<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RssReaderClass</class>
 <widget class="QMainWindow" name="RssReaderClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RssReader</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Resources/RssReader.ico</normaloff>Resources/RssReader.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="QSplitter" name="horizSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeWidget" name="feedTree">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::InternalMove</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::MoveAction</enum>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="itemsExpandable">
        <bool>false</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">Feeds</string>
        </property>
       </column>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QFrame" name="feedTitleFrame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QFrame#feedTitleFrame {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 200, 255, 150), stop:1 rgba(0, 0, 200, 175));
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="leftMargin">
            <number>2</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>2</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>6</number>
             </property>
             <item>
              <widget class="QLabel" name="feedImageLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="PyRssReader.qrc">:/RssReader/Resources/RssReader.ico</pixmap>
               </property>
               <property name="scaledContents">
                <bool>false</bool>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="feedNameLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>14</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>Feed Name</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QSplitter" name="vertSplitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTreeView" name="titleTree">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="showDropIndicator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <property name="animated">
            <bool>true</bool>
           </property>
           <attribute name="headerStretchLastSection">
            <bool>true</bool>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>915</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuExit">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionPreferences"/>
    <addaction name="actionOpen_Enclosure_Directory"/>
    <addaction name="separator"/>
    <addaction name="actionImport_OPML"/>
    <addaction name="actionExport_OPML"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuGroup_By">
    <property name="title">
     <string>Group By</string>
    </property>
    <addaction name="actionNo_Groups"/>
    <addaction name="actionDate"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionAbout_RssReader"/>
   </widget>
   <addaction name="menuExit"/>
   <addaction name="menuGroup_By"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_Feed"/>
   <addaction name="actionUpdate_Feeds"/>
   <addaction name="actionPurge_Old_News"/>
   <addaction name="actionCreate_Global_Filter"/>
   <addaction name="actionEdit_Language_Filter"/>
   <addaction name="actionEdit_Ad_Filter"/>
   <addaction name="actionAdd_to_Pocket"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionUpdate_Feeds">
   <property name="text">
    <string>Update Feeds</string>
   </property>
  </action>
  <action name="actionAdd_Feed">
   <property name="text">
    <string>Add Feed</string>
   </property>
  </action>
  <action name="actionPurge_Old_News">
   <property name="text">
    <string>Purge Old News</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences...</string>
   </property>
  </action>
  <action name="actionNo_Groups">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>No Groups</string>
   </property>
  </action>
  <action name="actionDate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
  <action name="actionAbout_RssReader">
   <property name="text">
    <string>About RssReader...</string>
   </property>
  </action>
  <action name="actionOpen_Enclosure_Directory">
   <property name="text">
    <string>Open Enclosure Directory</string>
   </property>
  </action>
  <action name="actionCreate_Global_Filter">
   <property name="text">
    <string>Manage Global Filters</string>
   </property>
  </action>
  <action name="actionEdit_Language_Filter">
   <property name="text">
    <string>Edit Language Filter</string>
   </property>
  </action>
  <action name="actionExport_OPML">
   <property name="text">
    <string>Export OPML...</string>
   </property>
  </action>
  <action name="actionEdit_Ad_Filter">
   <property name="text">
    <string>Edit Ad Filter</string>
   </property>
  </action>
  <action name="actionImport_OPML">
   <property name="text">
    <string>Import OPML...</string>
   </property>
  </action>
  <action name="actionAdd_to_Pocket">
   <property name="icon">
    <iconset>
     <normaloff>Resources/pocket.png</normaloff>Resources/pocket.png</iconset>
   </property>
   <property name="text">
    <string>Add to Pocket</string>
   </property>
   <property name="toolTip">
    <string>Adds the current article to Pocket</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PyRssReader.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>RssReaderClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>371</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
